armi/reactor/blueprints/blockBlueprint.py

```diff
@@ -129,9 +129,7 @@ def construct(
             c._resolveLinkedDims(components)
 
         boundingComp = sorted(components.values())[-1]
-        b = self._getBlockClass(boundingComp)(
-            "Bxxx{0}".format(grids.AXIAL_CHARS[axialIndex])
-        )
+        b = self._getBlockClass(boundingComp)("Bxxx-{0:03d}".format(axialIndex))
```

Comment:
This seems a bit odd to have to replicate the naming behavior here and in `makeName()` above. It might be a good idea to make `makeName()` a static method, and have it put `xxx` in when `assemNum is None`. Then we can do `bClass = self._getBlockClass(boundingComponent); b = bClass(bClass.makeName(None, axialIndex))` or similar.

Reply:
it is odd.

Reply:
hard to make a static method though b/c it changes parameter assemNum.

Reply:
easy to argue that this isnt the right place for assemNum to be set in the first place, yeah? Should the assemNum setting happen in `add()` instead?